docs/kyma/04-03-cluster-installation.md

```diff
@@ -331,5 +318,61 @@ For MacOS, run:
     ```
 
 
+  </details>
+  <details>
+  <summary>
+  Gardener
+  </summary>
+
+Install Kyma on a [GKE](https://cloud.google.com/kubernetes-engine/) or [AKS](https://azure.microsoft.com/services/kubernetes-service/) cluster deployed through [Gardener](https://gardener.cloud/).
```

Comment:
GKE and AKS are Kubernetes services. Gardener deploys K8S directly on VMs provided by GCP or Azure. Please change the links.